app/views/groups/_attrs.html.haml

```diff
@@ -13,6 +13,8 @@
       = render_attrs(entry, :self_registration_link, :self_registration_notification_email)
     = render_attrs(entry, :type_name)
     = render_present_attrs(entry, *entry.used_attributes(:description))
+    - if FeatureGate.enabled?('groups.nextcloud')
+      = render_present_attrs(entry, :nextcloud_url)
```

Comment:
Wouldn't our `used_attributes` mechanism also apply here? Couldn't you add the nextcloud_url to the group's used_attrs only if the feature gate allows it? Are there no downsides to nextcloud_url not being in the used_attributes?

Reply:
Good point, I'll try that.

Reply:
The views are now free from this specific FeatureGate.

I think that in most cases we can and should restrict the FeatureGate to ruby-files.

Good catch.